src/main/kotlin/org/jitsi/jibri/api/xmpp/XmppApi.kt

```diff
@@ -111,6 +111,14 @@ class XmppApi(
                         disableCertificateVerification = config.trustAllXmppCerts
                     }
 
+                    // FLOSS PATCH, 2021-01-12
```

Comment:
A couple of comments here:
1. Please make the config property an `Int` (I know MucClientConfiguration expects a string, but that's awkward, so we can keep is isolated there).
2. Either include the port number in the log message, or remove the message (I'm prety sure it will get printed elsewhere)
3. Remove the "FLOSS PATCH" comment. 

It could be simplified to just `port = config.xmppPort.toString()`

Reply:
Hi, I've tried to change it to Int but then a bunch of errors arise... sorry, I'm a Sysadmin and I can code in some langs but Kotlin isn't one of them.

If you can change it easily, please, feel free to go ahead, otherwise I'll check again on another bank holiday... right now I'm dealing with this to build a new docker image supporting Jibri+Pulseaudio+xmppPort :'(

Thanks!

PS: OFC, feel free to drop any comments as well :-)